src/openrct2/ride/Vehicle.cpp

```diff
@@ -7657,8 +7686,7 @@ bool Vehicle::UpdateTrackMotionForwards(CarEntry* carEntry, const Ride& curRide,
         {
             if (track_progress == 32)
             {
-                vehicle_type = carEntry->ReversedCarIndex;
-                carEntry = Entry();
+                isReversed = !isReversed;
```

Comment:
This seems like it should be a different PR

Reply:
Reverted this change - was something I was testing in proof of concept before deciding to make PR